backend/app/routes/web/search_results.py

```diff
@@ -94,7 +94,7 @@ async def search_post(
                 )
                 res_json["response"]["docs"] = copy.deepcopy(new_docs)
             except (Exception,):  # pylint: disable=broad-except
-                print("Something goes wrong..")
+                logger.exception("Exception happened during related services extension")
```

Comment:
`except Exception as e `
Return `e` in the log

Not the biggest fan of such a broad catch. If possible except only concrete exceptions.